.travis.yml

```diff
@@ -1,3 +1,9 @@
 language: ruby
 rvm:
-  - 2.1.5
\ No newline at end of file
+  - 2.1.6
+before_install:
+  - |
+    if [ "$LUMOS" == "true" ]; then
```

Comment:
@jdoconnor  I agree. This is a shim until refactoring has reached a point where we don't need separate ENVs to run tests. One of my upcoming goals is to introduce a series of mock objects for API calls.